File: VB/SeriesPointsSorting/Form1.vb
Imports DevExpress.XtraCharts
Imports System
Imports System.Windows.Forms

Namespace SeriesPointsSorting
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        #Region "#SeriesPointSorting"
        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
            Me.gSPTableAdapter.Fill(Me.gspDataSet.GSP)

            Dim seriesTemplate As SeriesBase = chart.SeriesTemplate
            seriesTemplate.SeriesPointsSorting = SortingMode.Ascending
            seriesTemplate.SeriesPointsSortingKey = SeriesPointKey.Value_1
        End Sub
        #End Region ' #SeriesPointSorting
    End Class
End Namespace

File: CS/SeriesPointsSorting/Form1.cs
using DevExpress.XtraCharts;
using System;
using System.Windows.Forms;

namespace SeriesPointsSorting {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        #region #SeriesPointSorting
        private void Form1_Load(object sender, EventArgs e) {
            this.gSPTableAdapter.Fill(this.gspDataSet.GSP);

            SeriesBase seriesTemplate = chart.SeriesTemplate;
            seriesTemplate.SeriesPointsSorting = SortingMode.Ascending;
            seriesTemplate.SeriesPointsSortingKey = SeriesPointKey.Value_1;
        }
        #endregion #SeriesPointSorting
    }
}
